docs/add_test.md

```diff
@@ -108,3 +108,21 @@ different places that are dynamically mapped in to the system at runtime:
 ## Test Development
 
 TODO: write note about hold_tests and test development
+
+Integration Testing Flow 
+
+1. Commit
+
```

Comment:
@ktredstone This whole process could be shortened by simply changing the test_X.out to what you would expect the test to be and using Travis to test it for you - if that makes sense?

You don't necessarilly _need_ to run text_aux.sh, but of course you do need to _modify_ test_aux.sh to get stuff done. It won't work in all cases but it might save some time here and there.

If that makes sense?

Also if you could make these steps any clearer it'll be great for the 'next dev'